addon/components/inputs/datetime.js

```diff
@@ -22,6 +22,18 @@ export default AbstractInput.extend({
 
   // == Computed Properties ====================================================
 
+  @readOnly
+  @computed('cellConfig.renderer.options.allowBlank')
+  allowBlank (allowBlank) {
```

Comment:
Is `allowBlank` really necessary? One would assume that if a default wasn't automatically set, it would be blank. I'm trying to keep this consistent with what was done to the `date` renderer which only added `defaultToCurrentDate` to the list of options.

Reply:
My thinking behind `allowBlank` was that the current behavior is to default to the current date, and not allow a blank value. I wanted to preserve the default behavior as much as possible, so by adding `allowBlank` I thought that existing code would be unaffected since none of it would have that option. Then I added `defaultToCurrentDateTime` to be used in conjunction so that people could still use the current default if they wanted to. 

Now that you mention it though, if there was just one setting `defaultToCurrentDateTime` and it was true by default (for backward compatibility), then people could set it to false if they wanted the field to be blank. If you think just having one property instead of two is better in this case, I will make the change.

Reply:
I do. It's also much simpler to keep track of and consistent with changes to other date components.